src/analyses/varEq.ml

```diff
@@ -119,6 +119,7 @@ struct
       | _ -> false
 
   (* kill predicate for must-equality kind of analyses*)
+  (* TODO: why unused? how different from below? *)
```

Comment:
They seem to be virtually identical, except the one below being a fair bit more chaotic

Reply:
I guess it might have been copied there, but without completely rewriting this all, I'd rather not touch this analysis more than necessary.